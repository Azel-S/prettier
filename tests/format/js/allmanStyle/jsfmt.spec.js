run_spec(__dirname, ["babel", "flow", "typescript"], { allmanStyle: true }); 