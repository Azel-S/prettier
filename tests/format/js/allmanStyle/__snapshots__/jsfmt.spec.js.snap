// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`allmanStyle.js - {"allmanStyle":true} format 1`] = `
====================================options=====================================
allmanStyle: true
parsers: ["babel", "flow", "typescript"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
if (true) { code; } 
else { code; }


if (cond) {
    stuff;
  } /* comment */ else if (cond) {
    stuff;
  }
  // comment
  else {
    stuff;
  }

for (;;) {
    code;
    /* comment
    comment c
    comment 
    */
    if (cond) {
        code;
    }
}
=====================================output=====================================
if (true)
{
  code;
} else
{
  code;
}

if (cond)
{
  stuff;
} /* comment */ else if (cond)
{
  stuff;
}
// comment
else
{
  stuff;
}

for (;;)
{
  code;
  /* comment
    comment c
    comment 
    */
  if (cond)
  {
    code;
  }
}

================================================================================
`;
